util/trace_replay.h

```diff
@@ -57,6 +60,7 @@ class Tracer {
 
   Status Write(WriteBatch* write_batch);
   Status Get(ColumnFamilyHandle* cfname, const Slice& key);
+  Status Iter(const uint32_t& cf_id, const Slice& key);
```

Comment:
This name is very confusing. Can we call it IteratorSeek() or something like that?